templates/federated_reporting/config.sh.mustache

```diff
@@ -36,6 +36,8 @@ CFE_FR_TABLES="${CFE_FR_TABLES:-$DEFAULT_TABLES}"
 # dump phase
 CFE_FR_COMPRESSOR="{{compressor}}"
 CFE_FR_COMPRESSOR="${CFE_FR_COMPRESSOR:-gzip}"
+CFE_FR_DECOMPRESSOR="{{decompressor}}"
+CFE_FR_DECOMPRESSOR="${CFE_FR_DECOMPRESSOR:-zcat}"
```

Comment:
I'd prefer a `CFE_FR_DECOMPRESS_ARG` and use the `CFE_FR_COMPRESSOR CFE_FR_DECOMPRESS_ARG` to decompress things.

Reply:
yeah, I didn't love this naming. :+1: will make a fix for it.